src/Compilers/Core/Portable/Operations/OperationFactory.cs

```diff
@@ -20,10 +20,11 @@ public static VariableDeclaration CreateVariableDeclaration(ImmutableArray<ILoca
                 semanticModel,
                 syntax,
                 type: null,
-                constantValue: default(Optional<object>));
+                constantValue: default(Optional<object>),
+                isImplicit: initialValue.IsImplicit);
```

Comment:
Why do we mark declaration as implicit when initial value is implicit? Isn't declaration always explicit?